satpy/readers/fci_l1c_fdhsi.py

```diff
@@ -181,54 +216,77 @@ def get_area_def(self, key, info=None):
         self.area = area
         return area
 
-    def calibrate(self, data, key):
+    def calibrate(self, data, key, measured, root):
         """Data calibration."""
 
-        # logger.debug('Calibration: %s' % key.calibration)
-        logger.warning('Calibration disabled!')
         if key.calibration == 'brightness_temperature':
-            # self._ir_calibrate(data, key)
-            pass
+            data = self._ir_calibrate(data, measured, root)
         elif key.calibration == 'reflectance':
-            # self._vis_calibrate(data, key)
-            pass
+            data = self._vis_calibrate(data, measured)
         else:
-            pass
+            raise RuntimeError("Reached unreachable code!")
```

Comment:
Could you change this to say that the calibration is unknown? Or something about not knowing how to calibration for calibration 'X'?

Reply:
Changed in c7161c0